scripts/initiate_ritual.py

```diff
@@ -38,8 +38,8 @@
 @click.option(
     "--fee-model",
     "-f",
-    help="The name of a fee model contract.",
-    type=click.Choice(FEE_MODELS),
+    help="The address of the subscription fee model contract.",
```

Comment:
```suggestion
    help="The address of the fee model/subscription contract.",
```
At some point we should resolve terminology re: subscription vs feeModel. In this particular case, the option should emphasize it's a fee model.